.env.example

```
@@ -2,24 +2,50 @@
+# OpenAI
+OPENAI_API_KEY=sk-proj-...
 
+# Azure OpenAI (TEC Agent resource)
+AZURE_OPENAI_API_KEY=your-azure-openai-key
+AZURE_OPENAI_ENDPOINT=https://tecagent.openai.azure.com/
+
+# xAI (Grok)
+XAI_API_KEY=xai-...
+
+# GitHub Models
+GITHUB_TOKEN=ghp_... or github_pat_...
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)